domain/src/main/java/tachiyomi/domain/manga/interactor/NetworkToLocalManga.kt

```diff
@@ -32,4 +33,14 @@ class NetworkToLocalManga(
     private suspend fun insertManga(manga: Manga): Long? {
         return mangaRepository.insert(manga)
     }
+
+    private suspend fun Manga.updateManga(id: Long) {
+        mangaRepository.update(
+            toMangaUpdate()
```

Comment:
Won't this override other fields? But i guess that's not an issue for non favorite manga?

Reply:
Any SManga field with non-null data will overwrite DB data. For non-favorite entries, it's will be fine, just like fresh browsing.

Other fields like reading mode, updated date... won't be affected